lib/ansible/modules/database/postgresql/postgresql_user.py

```diff
@@ -292,7 +292,7 @@ def user_alter(cursor, module, user, password, role_attr_flags, encrypted, expir
         # Do we actually need to do anything?
         pwchanging = False
         if password is not None:
-            if encrypted:
+            if encrypted == 'ENCRYPTED':
                 if password.startswith('md5'):
```

Comment:
@pilou- totally agree, I don't really like it, but I didn't wanna further complicate the PR, happy to change it though